View/SystemInfoViewBuilder.php

```diff
@@ -37,8 +38,15 @@ public function buildView(array $parameters)
     {
         $collector = $this->getCollector($parameters['systemInfoIdentifier']);
         $view = new SystemInfoView(null, [], $parameters['viewType']);
-        $view->setInfo($collector->collect());
 
+        try {
+            $collectedData = $collector->collect();
+        } catch (\Exception $e) {
```

Comment:
I'd introduce an interface for those exceptions, so we do not try to catch **all** exceptions here.

Reply:
Went for an empty one, feel free to propose something more suitable if you think it's not enough for generalization.